moonshot/src/runs/run_progress.py

```diff
@@ -108,7 +112,7 @@ def notify_progress(self, **kwargs) -> None:
                 RunProgress.sql_update_run_record,
             )
         else:
-            print(
+            logger.warning(
```

Comment:
if it fails to update the database, should this be error instead? should we change db_instance to database instance if we want to make the message more user friendly?

Reply:
agreeable with the database instance rename to be more user friendly. but the code didnt actually fail because the database instance does not exist. so this part im unable to say that it is an error